frontend/src/app/pacientes/models/dropdowns/opcoes-de-DDD.ts

```diff
@@ -0,0 +1,274 @@
+import { OpcaoCombo } from './opcao-combo';
+import { OPCAO_SELECIONE } from './opcao-selecione';
+
+export const OPCOES_DE_DDD: OpcaoCombo[] = [
```

Comment:
Se for colocar o DDD como lista de opções, a mesma deve ser extensível e ser mantida no banco de dados.